Pod/Core/StringGherkinExtension.swift

```diff
@@ -48,11 +48,11 @@ public extension String {
      */
     var humanReadableString: String {
         get {
-            guard case let c = self.characters where c.count > 1,
+            guard case let c = self.characters , c.count > 1,
```

Comment:
Has the `where` keyword been removed in Swift 3? It's a shame, I found `let x where y` more readable than this